zulipterminal/api_types.py

```diff
@@ -568,8 +568,8 @@ class ServerSettings(TypedDict):
     # Added in Zulip 2.1.0
     external_authentication_methods: List[ExternalAuthenticationMethod]
 
-    # TODO Refactor ZFL to default to zero
-    zulip_feature_level: NotRequired[int]  # New in Zulip 3.0, ZFL 1
+    # Refactor ZFL to default to zero DONE
+    zulip_feature_level: int  # New in Zulip 3.0, ZFL 1
```

Comment:
For TODO/FIXME comments, these can be removed when complete.

Note that `NotRequired` is related to the typing, namely that it may not be present in the object (dict), which remains true until 3.0, so should remain.

For reference, we don't yet have a type for initial_data, which is where the model is obtaining the feature level from at run-time; this location is for unauthenticated pre-fetch of the server information, some of which we use when generating a user zuliprc for them.

Reply:
To clarify, as part of this change it is fine to remove this comment entirely. That is, this PR should end up changing our default to zero.